src/payment/strategies/afterpay/afterpay-payment-strategy.ts

```diff
@@ -125,4 +134,24 @@ export default class AfterpayPaymentStrategy implements PaymentStrategy {
             return 'AU';
         }
     }
+
+    private _outsideLimits(amount: number, paymentMethod?: PaymentMethod): boolean {
+        const $minimum = paymentMethod?.initializationData?.min || 0;
+
+        return amount < $minimum || amount > paymentMethod?.initializationData?.max;
+    }
+
+    private async _loadPaymentMethod(gatewayId: string, methodId: string, options?: RequestOptions) {
```

Comment:
When purchasing outside the Afterpay limits, bigpay will not return Afterpay while fetching the paymentMethods, we want to catch that error and replace it with an `_invalidAmountErrorMessage`, any other error will be re threw.